src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-UI-Strings.json

```diff
@@ -71,6 +71,18 @@
       "At the end of turn 7, #ySnap. NL (It is turn #b%1$s now.)"
     ]
   },
+  "stsjorbsmod:ThirstOption": {
+    "TEXT": [
+      "Thirst",
+      "Consume wrath from an attack, heal 5 hp, and gain 1 max hp per wrath",
```

Comment:
I feel like this phrasing is a little ambiguous about whether the heal 5 is per-stack or not. Maybe something like `Consume ALL Wrath from a card. For each Wrath consumed, heal 5 HP and gain 1 Max HP.`?